Python/Product/PythonTools/ptvsd/repl/jupyter_client.py

```diff
@@ -330,54 +330,53 @@ def flush(self):
         pass
 
     def __shell_threadproc(self, client):
-        try:
-            last_exec_count = None
-            on_replies = self.__on_reply
-            while not self.exit_requested:
-                while self.__cmd_buffer and not self.exit_requested:
-                    cmd = self.__cmd_buffer.pop(0)
-                    if cmd.startswith('#nohistory'):
-                        self.__exec(cmd)
-                    else:
-                        self.run_command(cmd)
-                if self.exit_requested:
-                    break
-
-                try:
-                    m = Message(client.get_shell_msg(timeout=0.1))
-                    msg_id = m.msg_id
-                    msg_type = m.msg_type
-
-                    print('%s: %s' % (msg_type, msg_id))
-
-                    exec_count = m.content['execution_count', None]
-                    if exec_count != last_exec_count and exec_count is not None:
-                        last_exec_count = exec_count
-                        exec_count = int(exec_count) + 1
-                        ps1 = 'In [%s]: ' % exec_count
-                        ps2 = ' ' * (len(ps1) - 5) + '...: '
-                        self.send_prompt('\n' + ps1, ps2, allow_multiple_statements=True)
-
-                    parent_id = m.parent_header['msg_id', None]
-                    if parent_id:
-                        on_reply = on_replies.pop((parent_id, msg_type), ())
-                        for callable in on_reply:
-                            callable(m)
-                except Empty:
-                    pass
+    try:
```

Comment:
The indentation seems wrong? At least github is showing it that way

Reply:
The try is at the same level as the 'def'.

How did you test that this worked?

Reply:
oops. maybe I messed it up

Reply:
I was just manually testing

Reply:
How did it work if the indent was off?